### ISSUE TITLE
link to current version of book is broken.

### ISSUE THREAD

**Issue by nakshay** [rust-lang/unsafe-code-guidelines#103]
link in readme.md is broken ()
https://rust-rfcs.github.io/unsafe-code-guidelines/book

**Comment by gnzlbg** [comment]
Duplicate of #1 - this is blocked on adding a `GITHUB_TOKEN` for the repo.
